cmd/mysterium_client/mysterium_client.go

```diff
@@ -7,19 +7,21 @@ import (
 )
 
 func main() {
-	cmd := command_run.NewCommand()
 
 	options, err := command_run.ParseArguments(os.Args)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
 
-	if err := cmd.Run(options); err != nil {
+	cmd, err := command_run.NewCommand(options)
+	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
 
+	cmd.Run()
```

Comment:
Run without errors?

Reply:
cm.Run() does not return any errors, all errors are catched on command creation.